packages/base/src/asset-registries/Icons.ts

```diff
@@ -138,6 +139,37 @@ const getIconData = async (name: string) => {
 	return registry.get(registryKey);
 };
 
+/**
+ * Returns the accessible name for the given icon,
+ * or undefined if accessible name is not present.
+ *
+ * @param { string } name
+ * @return { Promise }
+ */
+const getIconAccessibleName = async (name: string) => {
```

Comment:
let's define the return type like `Promise<string | undefined>`

Reply:
done